glossary.asciidoc

```diff
@@ -94,7 +94,7 @@ TODO: Change for Clarity
 
 //
 Gas::
-    A virtual currency used in Ethereum to constrain computation of smart contracts. The Ethereum Virtual Machine uses an accounting mechanism to measure the consumption of gas and constrain (limit) the consumption of computing resources. See Turing-Complete.
+    A virtual fuel used in Ethereum to execute smart contracts. The Ethereum Virtual Machine uses an accounting mechanism to measure the consumption of gas and constrain (limit) the consumption of computing resources. See Turing-Complete.
```

Comment:
Agree. Maybe:
"A virtual fuel used in Ethereum to execute smart contracts. The Ethereum Virtual Machine uses an accounting mechanism to measure the consumption of gas and constrain (limit) the consumption of computing resources. Since transactions calling contracts must pay for gas and have an explicit gas limit, Ethereum prevents infinite-loops conditions and incentivizes contract developers to write efficient code. See Turing-Complete."

Reply:
:+1:

Reply:
Same question here, I am still learning best practices in GitHub.
How does one edit a commit? With another commit?
The last proposal was:
"A virtual fuel used in Ethereum to execute smart contracts. The Ethereum Virtual Machine uses an accounting mechanism to measure the consumption of gas and constrain (limit) the consumption of computing resources. Since transactions calling contracts must pay for gas and have an explicit gas limit, Ethereum prevents infinite-loops conditions and incentivizes contract developers to write efficient code. See Turing-Complete."